tests/runtime/main.rs

```diff
@@ -264,6 +264,7 @@ fn tests(name: &str, dir_name: &str) -> Result<Vec<PathBuf>> {
         };
 
         if !output.status.success() {
+            println!("dir: {}", out_dir.display());
```

Comment:
Is this added for debugging purposes?

Reply:
Well yes, but everything in that code block is for debugging purposes so this just makes the output more verbose. I can reverse it if you want, but then I'd just open a new PR adding it again 😀

Reply:
I am fine with checking this in!